src/GitInfo.macro.js

```diff
@@ -1,23 +1,42 @@
-const { execSync } = require('child_process');
 const { createMacro } = require('babel-plugin-macros');
+const { execSync } = require('child_process');
+
+const gitLogToJSON = (() => {
+    let message = '';
+    let refs = '';
+    const commit = {};
+    // only the commit message can have multiple lines. Make sure to always add at the end:
+    const logResult = execSync(`git log --format="%D%n%h%n%H%n%cI%n%B" -n 1 HEAD`)
```

Comment:
```suggestion
  const logResult = execSync('git log --format="%D%n%h%n%H%n%cI%n%B" -n 1 HEAD')
```
Unnecessary template string